CIPs/cip-0032.md

```diff
@@ -0,0 +1,117 @@
+---
+cip: 32
+title: Attestation Incentives
+author: Cody Born <cody@clabs.co>
+discussions-to: <URL>
+status: Draft
+type: Standards
+category: Ring 0
+created: 2020-12-11
+license: Apache 2.0
+---
+
+
+## Simple Summary
+
+Encourage liveness and global coverage of the Attestation Service (AS) through the incentivization of successful customer onboarding. This CIP will introduce slashing opportunities in the case of long term outages of the AS.
+
+## Abstract
+
+The Attestation Service performs a critical role in user onboarding when linking a phone number to a Celo address. When AS nodes are down or have poor coverage they lead to a degraded user experience. This proposal focuses on explicit incentives to run a highly available AS.
+
+## Motivation
+
+Despite Validator uptime on Celo being phenominal, the maintenance of the AS nodes are [sometimes neglected](https://metabase.celo-networks-dev.org/public/dashboard/b0a27650-1d62-4645-81d7-26ff7546ff0d?date_filter=past2weeks~). Today the implicit motivation to run an AS is to contribute to the health of the network which can help procure more votes from those that are interested in the success of Celo. This proposal attempts to make the incentive more explicit by introducing a slashing penalty in the case of prolonged AS downtime.
+
+In order to slash effectively, we will need an on-chain measure of AS availability. We're researching ways to provide this measurement in a manner that ensures validators are correctly rewarded when their AS is healthy and correctly punished when their AS is down. Until we have a large number of users, it's difficult to isolate a signal we can be confident in as a community. Therefore, we're starting with a conservative approach of measuring a minimum completion threshold over a period of time.
+
+## Specification
+
+- `B` is the number of consecutive blocks in which an elected Asstestation Service is required to complete at least one attestation.
```

Comment:
Just a nit but I'm not the biggest fan of super short variable names, especially in specs